requests/sessions.py

```diff
@@ -544,80 +552,90 @@ def request(self, method, url,
         return resp
 
     def get(self, url, **kwargs):
-        r"""Sends a GET request. Returns :class:`Response` object.
+        r"""Sends a GET request.
 
         :param url: URL for the new :class:`Request` object.
+        :param params: (optional) Dictionary or bytes to be sent in the query 
+            string for the :class:`Request`.
         :param \*\*kwargs: Optional arguments that ``request`` takes.
+        :return: :class:`Response <Response>` object
         :rtype: requests.Response
         """
 
-        kwargs.setdefault('allow_redirects', True)
```

Comment:
This line is what motivated me to this pull request. I removed this line here and in `options`, as its already the default.